### PR TITLE
Switch also to slow mode when LinqToDBConvertException occures (to di…

### PR DESCRIPTION
**Pull request by jogibear9988**
…splay the column)

### CODE REVIEW

**Comment by jogibear9988** [comment]
Atm you could not see when you get a LinqToDBConvertException for wich column.

What is a better fix? Maybe I should rethrow the LinqToDBConvertException and extend it with column name in the slow mapping? So Code relying on the LinqToDBConvertException would still work?

**Comment by sdanyliv** [comment]
At first, if you create PR, provide description and tests. From your description and code i see nothing.
Also you have break whole mapping system.
If you want to know which column fail - it is not possible without slowing down whole mapping.

**Comment by jogibear9988** [comment]
@sdanyliv I've to look how I could force the error...

If I only do the get of the Column Name when the Exception occurs, why do I slow it down?

File: Source/LinqToDB/Linq/QueryRunner.cs [hunk 1]
```diff
@@ -72,27 +72,7 @@ public T Map(IQueryRunner queryRunner, IDataReader dataReader)
...
 					_isFaulted = true;
-
-					return _mapper(queryRunner, dataReader);
-				}
-				catch (InvalidCastException ex)
-				{
-					if (_isFaulted)
-						throw;
-
-					if (DataConnection.TraceSwitch.TraceInfo)
```
[the remark this answers is not in the document]
Comment (jogibear9988) [reply Source/LinqToDB/Linq/QueryRunner.cs]: It's doubled Code. Look at the lines Github does not show (between 61 and 72)
I changed to conditional Catch clause

[the remark this answers is not in the document]
Comment (igor-tkachev) [reply Source/LinqToDB/Linq/QueryRunner.cs]: Is this compatible to all the platforms we support?

[the remark this answers is not in the document]
Comment (jogibear9988) [reply Source/LinqToDB/Linq/QueryRunner.cs]: It's only a compiler feature (C# 6). https://dailydotnettips.com/conditional-exception-handling-exception-filters-in-c-6-0/

File: Source/LinqToDB/LinqToDBException.cs [hunk 2]
```diff
@@ -13,6 +13,11 @@ namespace LinqToDB
 	[Serializable]
 	public class LinqToDBException : Exception
 	{
+		/// <summary>
+		/// The Column Name on which the Exception occurs
+		/// </summary>
+		public string ColumnName { get; set; }
```
Comment (sdanyliv) [inline Source/LinqToDB/LinqToDBException.cs]: @jogibear9988, do you need this information somewhere?

**Comment by sdanyliv** [comment]
@jogibear9988, Why not add `ColumnName` only to `LinqToDBConvertException` ?
`LinqToDBException` is too common class.

**Comment by jogibear9988** [comment]
I thought maybe we could use it for other scrnarios too. I could change!

**Comment by sdanyliv** [comment]
@jogibear9988, from my opinion it is much better to keep API more strict. If it is not critical, please move this property to appropriate class. 
Thanks.

**Comment by jogibear9988** [comment]
But in ConvertFromDataReaderExpression Line 168, I also added the Info to the LinqToDbException

**Pull request merged**